ask21-set.xml

```diff
@@ -368,6 +371,22 @@ See AUTHORS.txt for credits
             </winch>
         </hitches>
     </sim>
+    <limits>
+        <max-spoiler-extension-speed>135</max-spoiler-extension-speed>
```

Comment:
I was referring to http://www.segelfluggruppe-isartal.de/der-verein/die-flotte/ask21/ ("Sturzflugbremsen"), but probably a type error there ;)